integration_tests/macros/plugins/sqlserver/prep_external.sql

```diff
@@ -1,35 +1,71 @@
 {% macro sqlserver__prep_external() %}
 
     {% set external_data_source = target.schema ~ '.dbt_external_tables_testing' %}
-    
-    {% set create_external_data_source %}
-        IF NOT EXISTS ( SELECT * FROM sys.external_data_sources WHERE name = '{{external_data_source}}' )
-
-        CREATE EXTERNAL DATA SOURCE [{{external_data_source}}] WITH (
-            TYPE = HADOOP,
-            LOCATION = 'wasbs://dbt-external-tables-testing@dbtsynapselake.blob.core.windows.net'
-        )
-    {% endset %}
-
-    {% set external_file_format = target.schema ~ '.dbt_external_ff_testing' %}
-
-    {% set create_external_file_format %}
-        IF NOT EXISTS ( SELECT * FROM sys.external_file_formats WHERE name = '{{external_file_format}}' )
-
-        CREATE EXTERNAL FILE FORMAT [{{external_file_format}}] 
-        WITH (
-            FORMAT_TYPE = DELIMITEDTEXT, 
-            FORMAT_OPTIONS (
-                FIELD_TERMINATOR = N',', 
-                FIRST_ROW = 2, 
-                USE_TYPE_DEFAULT = True
+
+    {% if target.name == "synapse"%}
```

Comment:
This feels like another good reason to have adapters with different names. Since `prep_external` is just a hack for our integration tests anyway, I'm fine with it here